core/models/delay_log_normal.go

```diff
@@ -14,11 +15,8 @@ import (
 type ResponseDelayLogNormal struct {
 	UrlPattern     string         `json:"urlPattern"`
 	HttpMethod     string         `json:"httpMethod"`
-	Min            int            `json:"min"`
```

Comment:
lets keep internal struct as it is. It's easier to make it change in the future if these structs are not coupled with the simulation view.